### PR TITLE
SPCTR-3463: Fix - FAQ block content default padding issue in editor.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description
FAQ content default padding in editor is different from the padding in frontend.
https://brainstormforce.atlassian.net/browse/SPCTR-3463

### Screenshots

https://d.pr/v/Rmxam8

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,6 +222,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Fix: Forms - Added note for Forms block duplicate name fields to have distinct labels.
 * Fix: Global - Resolved an issue in which the Spectra blocks were unable to be previewed in the widget editor customizer.
 * Fix: Buttons - Resolved an issue in which the margins added to the button group would get overridden by child button margins.
+* Fix: FAQ - Resolved an issue in which the default content padding is not same in editor as in frontend.
```
Comment (bsf-zanev) [inline README.md:225]:
```suggestion
* Fix: FAQ - Resolved an issue in which the default content padding would appear differently in the editor and the front-end.
```

**Review by bsf-zanev — APPROVED** [review]
LGTM - Just update the readme and ensure that Backward Compatibility works as intended.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

**Review by adi3890 — APPROVED** [review]

**Comment by adi3890** [comment]
@bsf-zanev @rajmohanreddy774 
should we aslo add default values in attributes.php

**Comment by rajmohanreddy774** [comment]
> @bsf-zanev @rajmohanreddy774 should we aslo add default values in attributes.php

We already had the default values in attributes.php

**Review by adi3890 — APPROVED** [review]

**Pull request merged**